.github/pull_request_template.md

```diff
@@ -1 +1,12 @@
-**Please make sure that your pull request is targeting the `development` branch.**
+
+**Check list**:
+
+- [ ] Target the `development` branch
+- [ ] Use the `999-SNAPSHOT` version of Quarkus
+- [ ] Has tests (`mvn clean test`)
+- [ ] Works in native (`mvn clean package -Pnative`)
+- [ ] Has native tests (`mvn clean verify -Pnative`)
+- [ ] Link the documentation update pull request (if needed)
+- [ ] Updated or created the `README.md` file (with build and run instructions)
+- [ ] On new quickstarts, added the module to the root `pom.xml`
```

Comment:
And the root README.

Having the directory name being of the form `<component>-quickstart`.

Reply:
very good point! Adding it.